astropy/modeling/functional_models.py

```diff
@@ -288,7 +288,7 @@ class Gaussian2D(Fittable2DModel):
             \\Sigma = \\left(\\begin{array}{ccc}
             \\sigma_x^2               & \\rho \\sigma_x \\sigma_y \\\\
             \\rho \\sigma_x \\sigma_y & \\sigma_y^2
-            \end{array}\\right)
+            \\end{array}\\right)
```

Comment:
This docstring contains a lot of formulas but only one error so I've just fixed the error.

Will change it later.